radio/src/translations/de.h.txt

```diff
@@ -961,10 +961,10 @@
 #define TR_NO_SOUNDS_ON_SD     "Keine Töne" BREAKSPACE "auf SD"
 #define TR_NO_MODELS_ON_SD     "Kein Modelle" BREAKSPACE "auf SD"
 #define TR_NO_BITMAPS_ON_SD    "Keine Bitmaps" BREAKSPACE "auf SD"
-#define TR_NO_SCRIPTS_ON_SD    "Keine Scripte" BREAKSPACE "auf SD"
-#define TR_SCRIPT_SYNTAX_ERROR "Script Syntax Fehler"
-#define TR_SCRIPT_PANIC        "Script panic"   //??
-#define TR_SCRIPT_KILLED       "Script killed"  //??
+#define TR_NO_SCRIPTS_ON_SD    "Keine Skripte" BREAKSPACE "auf SD"
+#define TR_SCRIPT_SYNTAX_ERROR "Skript Syntaxfehler"
+#define TR_SCRIPT_PANIC        "Skript Panik"   //??
+#define TR_SCRIPT_KILLED       "Skript getötet"  //??

```

Comment:
The Translation is correctly but "getötet" must be refered to a person or an animal in german. So it makes no sense.